package.json

```diff
@@ -46,7 +55,10 @@
     "eslint-plugin-import": "^2.16.0",
     "eslint-plugin-prettier": "^3.0.1",
     "husky": "^1.3.1",
+    "jest": "^24.5.0",
     "prettier": "^1.16.4",
+    "ts-jest": "^24.0.1",
+    "ts-node": "^8.0.3",
```

Comment:
sure that it's merging issue, seems that `ts-node` is only `peerDependency` in current setup, could you check?

Reply:
Fixed. Thanks.